recipes/geos/all/patches/fix-cmake-3.9.0.patch

```diff
@@ -0,0 +1,41 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -19,7 +19,6 @@ set(DEFAULT_BUILD_TYPE Release)
+ # Require CMake 3.13+ with VS generator for complete support of VS versions
+ # and support by AppVeyor.
+ if(${CMAKE_GENERATOR} MATCHES "Visual Studio")
+-  cmake_minimum_required(VERSION 3.13 FATAL_ERROR)
+ endif()
+ 
+ list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
+@@ -282,12 +281,10 @@ endif()
+ #-----------------------------------------------------------------------------
+ # Benchmarks
+ #-----------------------------------------------------------------------------
+-add_subdirectory(benchmarks)
```

Comment:
Recently I discovered the `cmake.build(target="...")` which might help avoid this patch if we can specify the exact targets we need

Reply:
yes, but install target will build everything... that's the issue.

Reply:
Is this a bug 🤔 

https://github.com/conan-io/conan/blob/6e25cc80a5fb54ff7b318ecf5694ba6562aa9225/conans/client/build/cmake.py#L375

its running `cmake --build . --target=install` but it could be running `cmake --install . --target=myproject` 

CMake 2.X limitation?

Reply:
What do you mean? `install` is target on its own, it doesn't make sense to call `cmake --install . --target=myproject`

Reply:
It's a new feature in 3.15 after some digging https://stackoverflow.com/a/56457739/8480874

I don't have a setup to test but I recall it supporting the target method similar to the build, it's not in the documentation https://cmake.org/cmake/help/v3.19/manual/cmake.1.html#install-a-project

Reply:
feature is nice, maybe new CMake helper should learn about it